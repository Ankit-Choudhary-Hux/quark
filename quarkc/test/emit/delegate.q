quark 0.6.100;
// xfail:rb
void main(List<String> args) {
    print("This shadows ruby builtin module 'delegate'");
}
