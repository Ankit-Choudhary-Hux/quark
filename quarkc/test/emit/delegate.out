[one, two, 3]
ENCODED
PONG
